test/OpenTelemetry.Extensions.Tests/OpenTelemetry.Extensions.Tests.csproj

```diff
@@ -4,6 +4,7 @@
     <Description>Unit test project for OpenTelemetry .NET SDK preview features and extensions</Description>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
     <TargetFrameworks>net7.0;net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net48;net472;net471;net47;net462</TargetFrameworks>
```

Comment:
default `tfm-list` `'[ "net462", "net6.0", "net7.0" ]'`. I do not think that we should include more targets here, and in CI execute only subset.

Reply:
I had copied that list from [here](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/4db5ac689e2a5e5cf761d3df65d049829358af21/test/OpenTelemetry.Exporter.Geneva.Tests/OpenTelemetry.Exporter.Geneva.Tests.csproj#L8) but I don't feel passionately that it must be there so I switched it to just `net462`.

Should our CI actually run against all supported frameworks? 🤷

Reply:
I think that CI should execute all tests defined in project files. Otherwise it can be misleading.

Adding more TFMs can be handled separaterly.